### DISCUSSION TITLE
Prevent adjacent claims

### DISCUSSION THREAD

**Discussion by ShaneBeee** [GriefPrevention/GriefPrevention#124] [Ideas]
I have a simple suggestion! 
Maybe set up a configurable minimum distance you need to be from another claim to initially claim land.
For example, I set it to 100, then a player may not claim within 100 blocks of my claim. This would help keep players claims spread out, and stop players from running around other claims, and claiming land!

I see that this may be an issue when players WANT to claim next to each other! FOR example, a friend of mine and I have a little town going, so obviously we each have claims that are touching each other! 
Maybe set it so the only way you can claim next to another players claim, is if they "trust" you?!?

I know several servers have issues with people not listening to the rules (no surprise there) and they claim very very close to other claims. This would help cut that down!
Thanks for reading!

**Comment by zedwick** [comment]
> Maybe set it so the only way you can claim next to another players claim, is if they "trust" you?!?

I think a better solution would be to make it so only claim managers may claim directly next to the claim. If they're trying to use their combined claim blocks to secure an area like that, safe to assume they'd have that level of trust in all the claims.

**Comment by ShaneBeee** [comment]
That would make more sense.  I think that's what I meant 
/permissiontrust

I've had a few dummies on my server who purposely claim next to another player... which you know they're doing it to bother said player. 
I'd love to nip that in the bud without having to have more rules that people don't read/acknowledge/listen to!

**Comment by RoboMWM** [comment]
I know Big_Scary had some sort of comment on this. Either way - if you want that space, I don't see why players can't just claim that said space. I also see some other potential issues - claims that are diagonal (and thus not as "close" as one aligned to a side), and potential confusion on the offending claim.

This would also cause extra checks to be made every time a claim is created or resized.

However it could be worth mentioning in a message of some sort though - and it's usually the owner of the adjacent claim that creates a new claim due to not knowing/bothering to figure out how to extend their existing claim.
